plugins/confluence-plugin/src/components/PageContent.test.tsx

```diff
@@ -0,0 +1,101 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import PageContent from "./PageContent";
+
+const mockPageContent = {
```

Comment:
This looks like the same as the other mock. Assuming that's true, can/should we centralize it? This reduces file size, reduces likelihood of human error (updating and testing 1 place but not another), and makes future re-use easier.